### PR TITLE
[NayNay] Exposing substrate

### LINKED ISSUE #427: Expose substrate as part of the Entropy Class rather than the Entropy Instance
**Title**: `<A clear and concise title that reflects the nature of your issue.>`

**Issue type**: (Select one)

- [ ] Bug
- [ ] Question
- [ ] Improvement

**Description**:

Describe the issue you're encountering in detail.

**Steps to reproduce**:

Provide a step-by-step explanation of how to reproduce the issue.

**Error messages**:

Include any relevant error messages, logs, or screenshots.

**Expected behavior**: 

What were you expecting to happen? This seems like a silly question but it's very helpful to know what you'd were hoping to see. Please be specific.

**Additional information**:

- Operating System: (e.g., Windows 11, MacOS Monterey, Ubuntu 22.04)
- Other Software/Hardware: (if applicable)

### PR DESCRIPTION
**Pull request by rh0delta**
## Related Issue(s)

- Resolves #427 

## Proposed Changes

- added new utility method to create substrate api instance, without needing to create an entropy instance

## Testing

- [ ] Unit tests added/updated
- [ ] Integration tests added/updated
- [ ] Manual testing performed

## Checklist

- [ ] I have performed a self-review of my code.
- [ ] I have commented my code.
- [ ] I have updated documentation, where necessary.

### CODE REVIEW

File: src/utils/index.ts [hunk 1]
```diff
@@ -112,7 +113,17 @@ export async function sendHttpPost (url: string, data: any): Promise<any> {
...
 }
 
+/**
+ * Returns substrate api without needing to instantiate an entropy instance
+ * 
+ * @param {string} endpoint - Endpoint for the api
+ * @returns {ApiPromise} Api tool to interact with protocol
+ */
 
+export function getSubstrate (endpoint: string): ApiPromise {
```
Comment (rh0delta) [inline src/utils/index.ts:123]:
```suggestion
export function createSubstrate (endpoint: string): ApiPromise {
```

File: src/utils/index.ts [hunk 2]
```diff
@@ -112,7 +113,17 @@ export async function sendHttpPost (url: string, data: any): Promise<any> {
...
 }
 
+/**
+ * Returns substrate api without needing to instantiate an entropy instance
+ * 
+ * @param {string} endpoint - Endpoint for the api
+ * @returns {ApiPromise} Api tool to interact with protocol
+ */
 
+export function createSubstrate (endpoint: string): ApiPromise {
```
Comment (frankiebee) [inline src/utils/index.ts:123]: Let call this createSubstarte

File: src/utils/index.ts [hunk 3]
```diff
@@ -112,7 +113,17 @@ export async function sendHttpPost (url: string, data: any): Promise<any> {
   return oks
 }
 
+/**
+ * Returns substrate api without needing to instantiate an entropy instance
```
Comment (rh0delta) [inline src/utils/index.ts:117]:
```suggestion
 * Creates substrate api provider without needing to instantiate an entropy instance
```

**Comment by frankiebee** [comment]
I'd like a little more documentation for this pointing out the polkadot docs

**Comment by frankiebee** [comment]
> I'd like a little more documentation for this pointing at the polkadot docs to communicate that we don't maintain the underlying api for substrate and a warning that this might change in the future

will handle with this issu: #436

**Review by frankiebee — APPROVED** [review]
sweeet

**Pull request merged**